handsontable/src/editors/autocompleteEditor/__tests__/autocompleteEditor.spec.js

```diff
@@ -1054,94 +1053,216 @@ describe('AutocompleteEditor', () => {
       expect(editor.find('tbody td:eq(5)').text()).toEqual('6');
     });
 
-    it('should display the dropdown above the editor, when there is not enough space below the cell AND' +
-       'there is more space above the cell', async() => {
+    it('should display the dropdown above the editor, when there is not enough space below (table has defined size)', async() => {
       handsontable({
-        data: Handsontable.helper.createSpreadsheetData(30, 30),
-        columns: [
-          {
-            editor: 'autocomplete',
-            source: choices
-          }, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}
-        ],
+        data: createEmptySpreadsheetData(30, 30),
+        editor: 'autocomplete',
+        source: choices,
         width: 400,
-        height: 400
+        height: getDefaultRowHeight() * 9,
```

Comment:
what is `9` here?

Reply:
The tests - to be the same for each theme, needs to be set up with the same setup. In this case the viewport height is "theme default height" x 9.

Reply:
I've added code comment 94fc8a4849f478d4be40d17086cc5c29801cae49